### ISSUE TITLE
Offline payment gateway design update

### ISSUE THREAD

**Issue by ilyasfoo** [woothemes/woocommerce#52392]
### Issue Description

Follow-up to https://github.com/woocommerce/woocommerce/pull/52213

Figma: xO8YdpKfaVoP2d8NvweZ5I-fi-3106_63491

Pre-requisite: https://github.com/woocommerce/woocommerce/pull/52161

Currently, the design is a direct copy via [SCSS extend](https://github.com/woocommerce/woocommerce/blob/3efa429d9862d347438ab6ba0aa7a7ebe438049e/plugins/woocommerce/client/admin/client/settings-payments/settings-payments-offline.scss#L1-L6) of another component. Once the base design PR is done, we'd need to implement the proper design to match with Figma design. 
 
![Image](https://github.com/user-attachments/assets/c6d35b82-fc96-47fa-bc50-136453b3785c)

### Acceptance Criteria

Depending on what's done in the other PR, we need to address the following:

1. Full-width table
1. Full screen with white background on page body
1. Page title
1. Settings tab navigation
1. Direct bank transfer SVG icon (missed this with the previous PR)
1. Fonts

**Comment by anu-rock** [comment]
Hi @ilyasfoo 👋 I did a quick walkthrough of our Native Onboarding experience so far (p1730460458852349-slack-C03KTTK2YMA) and noted a couple of observations related to offline PMs:

> 4. Clicking the "Offline payment methods" link loads a new page instead of doing an SPA navigation.
> 5. The back link on an offline PM management page leads to the top-level Payments settings page instead of the Offline payment methods page. Besides, the back link is incosistent with the mobile-like back button used elsewhere in designs.

Keen to hear your thoughts and whether we can use this issue to fix them if necessary.

**Comment by ilyasfoo** [comment]
Hi, @anu-rock! Thanks for the feedback.

> Clicking the "Offline payment methods" link loads a new page instead of doing an SPA navigation.

Ideally it should work like that, but I'm unsure if it should be in scope of this issue. cc @adrianduffell if you have thoughts on when should we work on SPA navigation

> The back link on an offline PM management page leads to the top-level Payments settings page instead of the Offline payment methods page. Besides, the back link is incosistent with the mobile-like back button used elsewhere in designs.

Good catch! I'll add this to the issue. Thanks!

Edit: I'll create another issue instead since it can be worked in parallel

Edit: Issue created https://github.com/woocommerce/woocommerce/issues/52570

**Comment by anu-rock** [comment]
Thank you, @ilyasfoo 🙌

**Comment by adrianduffell** [comment]
> cc @adrianduffell if you have thoughts on when should we work on SPA navigation

@ilyasfoo, @anu-rock We'll have SPA navigation come about as part of the [settings modernization](https://github.com/woocommerce/woocommerce/issues/52278) work happening in parallel.
